packages/data-context/src/actions/WizardActions.ts

```diff
@@ -375,6 +378,61 @@ export class WizardActions {
   private ensureDir (type: 'component' | 'e2e' | 'fixtures') {
     return this.ctx.fs.ensureDir(path.join(this.projectRoot, 'cypress', type))
   }
+
+  private supportFileBody (fileName: 'e2e' | 'component', language: CodeLanguageEnum) {
+    return dedent`
+      // ***********************************************************
+      // This example support/${fileName}.${language} is processed and
+      // loaded automatically before your test files.
+      //
+      // This is a great place to put global configuration and
+      // behavior that modifies Cypress.
+      //
+      // You can change the location of this file or turn off
+      // automatically serving support files with the
+      // 'supportFile' configuration option.
+      //
+      // You can read more here:
+      // https://on.cypress.io/configuration
+      // ***********************************************************
+  
+      // Import commands.js using ES2015 syntax:
+      import './commands'
+  
+      // Alternatively you can use CommonJS syntax:
+      // require('./commands')
+    `
+  }
+
+  private commandsFileBody (language: CodeLanguageEnum) {
+    return dedent`
+      // ***********************************************
```

Comment:
If the language is typescript, Should this file include the example types for these custom commands as well?